src/styles/Grid/Star.js

```diff
@@ -0,0 +1,6 @@
+import styled from "styled-components";
+
+const Star = styled.div`
+    margin-top: 0px;
+`;
+export default Star;
```

Comment:
@filiptronicek Instead of creating new **styled component** we can just wrap `<Octicon />` in `div` tag